app/models/user.rb

```diff
@@ -42,4 +43,12 @@ def follow(another_user)
     Relationship.create(leader: another_user, follower: self) if can_follow?(another_user)
   end
 
+  def deactivated!
+    update_column(:active, false)
+  end
+
+  def active?
```

Comment:
This is not needed. Rails provides this method by default for every boolean column :)